src/andromede/simulation/output_values.py

```diff
@@ -44,17 +42,32 @@ class Variable:
         _value: Dict[TimeScenarioIndex, float] = field(init=False, default_factory=dict)
         _size: Tuple[int, int] = field(init=False, default=(0, 0))
 
+        ignore: bool = field(default=False, init=False)
+        rel_tol: float = field(default=1.0e-9, init=False)
+        abs_tol: float = field(default=0.0, init=False)
+
         def __eq__(self, other: object) -> bool:
```

Comment:
I think it would be more clear to have a separate method, so that == remains the strict equality, like for floats etc.

Also, it's not very clear which `rel_tol` and `abs_tol` get used between the LHS and RHS with this approach, it would be more clear with method arguments like in `pytest.approx`:
```python
def output_equals(output1, output2, rel: float, abs: float) -> bool:
```

Reply:
Note that we could also use features of `pytest`, but it would "break" the typing:
`pytest.approx()` can be used inside objects to be compared:
```python
def test_compare_floats():
    a = Foo(1, 3.00000001)
    b = Foo(1, pytest.approx(3.0, abs=1e-3))
    assert a == b
```
So we could assign approx float to our expected values in tests:
```python
test_output.component("component_id_test").var("component_var_name").value = pytest.approx(1.0, rel=0.1, abs=0.1)
```
or possibly provide a method to convert an "exact" class in an "approximated" class containing approximated floats instead of floats.

Reply:
I like this last idea of an approximated class for floats. I will try to implement something on this line this afternoon